src/components/ui/radio-group.tsx

```
@@ -0,0 +1,152 @@
+    const resolvedId = id ?? groupId;
+
+    const helperId = helperText ? `${resolvedId}-help` : undefined;
+    const errorId = error ? `${resolvedId}-error` : undefined;
+    const describedBy = error ? errorId : helperId;
+
+    const effectiveIntent = error ? "error" : intent;
+
+    return (
+      <div className="flex flex-col gap-2">
+        {label && (
+          <label className="text-label text-neutral-foreground">{label}</label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)